cmd/root.go

```diff
@@ -0,0 +1,83 @@
+package cmd
```

Comment:
why root.go and not main.go? What are you using as a reference for this directory/file structure?

Reply:
It's the one recommended in Cobra readme ("Using the Cobra Library"), for the root command ("tcpgoon"). If we go for a subcommands mode, we will create specific files for each one.

Reply:
I've rename root.go to tcpgoon.go as the main command. If we add more modes/subcommands later we will creates files for each one.